src/main/resources/application.conf

```diff
@@ -0,0 +1,13 @@
+sqs2kinesis {
+  sqs-queue = ${SQS_QUEUE}
```

Comment:
`?` is missing, `${?SQS_QUEUE}`

Reply:
`?` is removed on purpose. This was motivation: https://github.com/snowplow-incubator/sqs2kinesis/pull/4#pullrequestreview-389121868